docs/compatible_configs.md

```diff
@@ -0,0 +1,211 @@
+# _Black_ compatible configurations
+
+Most of the changes that _Black_ make are harmless, but a few do conflict against other
+tools. It is not uncommon to be using other tools alongside _Black_ like linters and
+type checkers. Some of them need a bit of tweaking to resolve the conflicts. Listed
+below are _Black_ compatible configurations for the commonly used tools out there with
+their explanations.
+
+## isort
+
+[isort](https://pypi.org/p/isort/) helps to sort and format imports in your Python code.
+_Black_ also formats imports, but in a different way from isort's defaults which leads
+to conflicting changes.
+
+_Black_ wraps imports that surpass `line-length` by moving identifiers into their own
+indented line. If that still doesn't fit the bill, it will put all of them in seperate
+lines and put a trailing comma. A more detailed explanation of this behaviour can be
+[found here](https://github.com/psf/black#how-black-wraps-lines).
+
+isort should be configured to match _Black_'s import formatting style by these options:
+
+- `multi_line_output = 3`
+- `include_trailing_comma = true`
+- `force_grid_wrap = 0`
+- `combine_as_imports = true`
+
+isort should be configured to wrap imports when they surpass _Black_'s default limit of
+88 characters via `line_length = 88`.
```

Comment:
I am not proud of this. It does a poor job of explaining the configuration. The problem is that I haven't used isort before and I don't know these options well enough to explain them well. Even reading the isort documentation didn't really do much for me.

Reply:
Well, after realizing my isort configuration was not ideal, I spent some more time reading the isort documentation and finally the options made sense. **Issue resolved.**